src/main/java/org/isf/lab/service/LabIoOperationRepository.java

```diff
@@ -45,4 +49,17 @@ public interface LabIoOperationRepository extends JpaRepository<Laboratory, Inte
 	List<Laboratory> findByLabDateBetweenAndExamDescriptionAndPatientCode(LocalDateTime dateFrom, LocalDateTime dateTo, String exam, Integer patient);
 	
 	Page<Laboratory> findByLabDateBetweenOrderByLabDateDesc(LocalDateTime dateFrom, LocalDateTime dateTo, Pageable pageable);
+	
+	@Query(value = "select lab from Laboratory lab where lab.labDate >= :dateFrom or lab.labDate < :dateTo")
+	Page<Laboratory> findByLabDateBetweenOrderByLabDateDescPage(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, Pageable pageable);
```

Comment:
There is not `order` clause and the JQL should be like the follow

```suggestion
	@Query(value = "select lab from Laboratory lab where lab.labDate >= :dateFrom and lab.labDate < :dateTo order by lab.labDate desc")
	Page<Laboratory> findByLabDateBetweenOrderByLabDateDescPage(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, Pageable pageable);
```